unit-tests/native/src/test/scala/scala/scalanative/unsafe/SizeofTest.scala

```diff
@@ -9,68 +9,78 @@ import scalanative.unsigned._
 class SizeofTest {
 
   @Test def testByte(): Unit = {
-    assertTrue(sizeof[Byte] == 1.toULong)
+    assertTrue(sizeof[Byte] == 1.toUSize)
```

Comment:
To minimize the diff, I was thinking of keeping it that way since it's similar on the main branch. I can follow up with another PR to fix this, but didn't want to clutter this PR any further.

Reply:
Ok, it can be changed in the follow-up. It's highly uncommon this tests would fail so it's not a high priority